tests/data/auto-concurrency-template.toml

```diff
@@ -0,0 +1,35 @@
+[params]
```

Comment:
I see this file is skipped, but I can't see how. Where does that happen?

Reply:
That file is one directory level up from the actual tests.

Reply:
Aha! I see now.